frontend/src/employee-frontend/components/unit/unit-reservations/ChildDayReservation.tsx

```diff
@@ -145,7 +153,7 @@ export default React.memo(function ChildDayReservation({
                   />
                 </>
               )}
-              {reservation.staffCreated && '*'}
+              {reservation?.staffCreated && '*'}
```

Comment:
Poista tarpeeton optional chaining käyttö
```suggestion
              {reservation.staffCreated && '*'}
```